src/js/heuristicblocking.js

```diff
@@ -519,7 +689,9 @@ function startListeners() {
     extraInfoSpec.push('extraHeaders');
   }
   chrome.webRequest.onBeforeSendHeaders.addListener(function(details) {
-    return badger.heuristicBlocking.heuristicBlockingAccounting(details);
+    badger.heuristicBlocking.heuristicBlockingAccounting(details);
+    badger.heuristicBlocking.pixelCookieShareListener(details);
```

Comment:
We don't need to call this if `heuristicBlockingAccounting` finds cookie tracking.

Reply:
We also still want all those checks at the top of `heuristicBlockingAccounting` (only third-party, don't learn if incognito, ...). So perhaps pixel cookie sharing checking should be called from `heuristicBlockingAccounting` but only coming from `onBeforeSendHeaders`.